comps/comps-katello-pulpcore-el7.xml

```diff
@@ -97,7 +97,8 @@
       <packagereq type="default">python3-urllib3</packagereq>
       <packagereq type="default">python3-whitenoise</packagereq>
       <packagereq type="default">python3-yarl</packagereq>
-      <packagereq type="default">python3-zipp</packagereq>
+	    <packagereq type="default">python3-zipp</packagereq>
```

Comment:
Looks like indenting went wrong here

Reply:
Interesting, I only see it in the raw.  I'll fix.